src/conditions/EqualsCondition.ts

```diff
@@ -26,8 +26,10 @@ export class EqualsCondition implements IConditionFunction {
 
         return Object.keys(args).every((key) => {
             return CommonUtil.matchesAnyElement(args[key], (elm) => {
-                return context[key] && ConditionUtil.getValueByPath(context, elm) === context[key];
+                const keyValue = key.startsWith('$.') ?  ConditionUtil.getValueByPath(context, key) : context[key];
```

Comment:
ConditionUtil.getValueByPath(context, key) -> this is going to give only key right. 
So shouldn't this be:
const keyValue = context[ConditionUtil.getValueByPath(context, key)];

Reply:
ConditionUtil.getValueByPath(context, key) will give value on context when key is in JSON path format, I need this so I can compare nested values, e.g.
```
{ '$.requester.employee.id': '$.instance.employee' }
```
or
```
{ '$.instance.creatorId': '$.instance.assigneeId' }
```

Reply:
can you please give a full example with grants and context and conditions?
Also please update: https://github.com/tensult/role-acl/blob/develop/test/acl.spec.ts for this use case.